api/drf_views.py

```diff
@@ -202,18 +211,30 @@ class RegionSnippetViewset(ReadOnlyVisibilityViewset):
     filter_class = RegionSnippetFilter
     visibility_model_class = RegionSnippet
 
+    def get_serializer_class(self):
+        if self.request.GET.get('tableau', 'false').lower() == 'true':
+            return RegionSnippetCsvSerializer
+        return RegionSnippetSerializer
+
+
 class CountrySnippetFilter(filters.FilterSet):
     country = filters.NumberFilter(field_name='country', lookup_expr='exact')
     class Meta:
         model = CountrySnippet
         fields = ('country',)
 
+
 class CountrySnippetViewset(ReadOnlyVisibilityViewset):
     authentication_classes = (TokenAuthentication,)
-    serializer_class = CountrySnippetSerializer
     filter_class = CountrySnippetFilter
     visibility_model_class = CountrySnippet
 
+    def get_serializer_class(self):
+        if self.request.GET.get('tableau', 'false').lower() == 'true':
```

Comment:
This is repetead many times. Maybe use a function `is_tableau`?
```python
def is_tableau(request):
    return request.GET.get('tableau', 'false').lower() == 'true'
```

Reply:
https://github.com/IFRCGo/go-api/pull/830/commits/08e9d0fff77ace5fa3b8dec55192c668a795693f :+1:

Reply:
Nice.
Quick Note: You can do `if is_tableau(self.request):` instead of `if is_tableau(self.request) is True:`.